travisci/install_dependencies.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+if [[ "$COVERALLS" = 'true' && ! "${TRAVIS_PERL_VERSION}" =~ '5.10' ]]
+then
+  awk '!/Devel::Cover/' ensembl/cpanfile > tmp_cpanfile && mv tmp_cpanfile ensembl/cpanfile
+  awk '!/Devel::Cover/' cpanfile > tmp_cpanfile && mv tmp_cpanfile cpanfile
```

Comment:
Would be less verbose with `sed`:
```suggestion
  sed -i '/Devel::Cover/d' ensembl/cpanfile
  sed -i '/Devel::Cover/d' cpanfile
```